test/sql/storage/compression/roaring/roaring_array_simple.test

```diff
@@ -0,0 +1,39 @@
+# name: test/sql/storage/compression/roaring/roaring_array_simple.test
+# description: Test bitpacking with nulls
+# group: [roaring]
+
+# This test defaults to another compression function for smaller block sizes,
+# because the bitpacking groups no longer fit the blocks.
+require block_size 262144
+
+load __TEST_DIR__/test_roaring.db
+
+statement ok
+PRAGMA force_compression='roaring'
+
+# simple compression with few values
+statement ok
+CREATE TABLE test (a BIGINT);
+
+statement ok
+INSERT INTO test SELECT case when i%25=0 then 1337 else null end FROM range(0,10000) tbl(i);
```

Comment:
Should probably add a variant of this that uses the uncompressed array container